packages/editor/src/components/post-author/check.js

```diff
@@ -35,14 +36,14 @@ export function PostAuthorCheck( {
 export default compose( [
```

Comment:
Nit: Could easily be transformed to hooks now.

Reply:
I guess this means we should remove the test for `PostAuthorCheck`. The component won't be really unit-testable after this change.

Reply:
Never mind, I found examples of how to update the tests.

Reply:
To be honest, these tests are not the best we have either :)